samples/pom.xml

```diff
@@ -18,10 +18,8 @@
 
   <modules>
     <module>build-cache</module>
-    <module>cargo-jolokia</module>
     <module>copy-from-container</module>
     <module>custom-net</module>
-    <module>data-jolokia</module>
```

Comment:
Why have we deleted these samples? Is it not possible to modify these samples to align with JDK 11?

Reply:
I tried with little success.  I'm not sure it's worth the effort.  What samples are developers interested in and will show off d-m-p's capabilities?  For example; Spring, Blade, Dropwizard, Micronaut, Quarkus, Jhipster, MicroProfile, Hilla

Reply:
@rohanKanojia, are you and @rhuss ok to remove these samples?

Reply:
Umm, okay. Let's remove these samples.